tools/appveyor.psm1

```diff
@@ -311,6 +311,12 @@ function Invoke-AppVeyorTest
         Write-Host -Foreground Green 'Running all CoreCLR tests..'
     }
 
+    # Remove telemetry semaphore file in CI
+    $TelemetrySemaphoreFilePath = join-path $env:CoreOutput DELETE_ME_TO_DISABLE_CONSOLEHOST_TELEMETRY
```

Comment:
We should have a separate issue for other scripts/modules.  For this file, we should just fix the casing on the new code.